contrib/systemd/nv-dhcp.service

```diff
@@ -0,0 +1,16 @@
+[Unit]
+Description=Netavark DHCP proxy
+Requires=nv-dhcp.socket
+After=nv-dhcp.socket
+StartLimitIntervalSec=0
+
+[Service]
+Type=exec
+Delegate=true
+Restart=always
+KillMode=process
```

Comment:
`KillMode=process` is being discouraged by systemd and will yield warnings. I recommend leaving the default which is "control-group" which will nuke everything in the cgroup of the service.  That should be enough even when the proxy forks or execs other processes.